### ISSUE TITLE
Maximum call stack size exceeded

### ISSUE THREAD

**Issue by realgsong** [NagRock/ts-mockito#82]
With 2.2.8, I got "Maximum call stack size exceeded" error and don't have any idea where to start investigation. 2.2.7 works successful BTW.
 
```
HeadlessChrome 0.0.0 (Linux 0.0.0) SubscribeComponent should create FAILED
	Failed: Maximum call stack size exceeded
	RangeError: Maximum call stack size exceeded
	    at Mocker.webpackJsonp.../../../../ts-mockito/lib/Mock.js.Mocker.createInstancePropertyDescriptorListener home/circleci/project/front/node_modules/ts-mockito/lib/Mock.js:96:1)
	    at Object.get home/circleci/project/front/node_modules/ts-mockito/lib/Mock.js:48:1)
	    at isUnconfigurable home/circleci/project/front/node_modules/zone.js/dist/zone.js:2185:1)
	    at Function.Object.defineProperty home/circleci/project/front/node_modules/zone.js/dist/zone.js:2148:1)
	    at Mocker.webpackJsonp.../../../../ts-mockito/lib/Mock.js.Mocker.createInstancePropertyDescriptorListener home/circleci/project/front/node_modules/ts-mockito/lib/Mock.js:99:1)
	    at Object.get home/circleci/project/front/node_modules/ts-mockito/lib/Mock.js:48:1)
	    at isUnconfigurable home/circleci/project/front/node_modules/zone.js/dist/zone.js:2185:1)
	    at Function.Object.defineProperty home/circleci/project/front/node_modules/zone.js/dist/zone.js:2148:1)
	    at Mocker.webpackJsonp.../../../../ts-mockito/lib/Mock.js.Mocker.createInstancePropertyDescriptorListener home/circleci/project/front/node_modules/ts-mockito/lib/Mock.js:99:1)
	    at Object.get home/circleci/project/front/node_modules/ts-mockito/lib/Mock.js:48:1)
```

Is there any known issues related with this?

**Comment by NagRock** [comment]
Sorry for this. I will try to fix this ASAP.

**Comment by SayingsOlly** [comment]
I will save the problem today, works fine before, and I didn't update the version.....

 `HeadlessChrome 63.0.3239 (Windows 8.1.0.0) AdminComponent should return correct select page FAILED
	Failed: Maximum call stack size exceeded
	RangeError: Maximum call stack size exceeded
	    at Proxy.hasOwnProperty (<anonymous>)
	    at Mocker.webpackJsonp../node_modules/ts-mockito/lib/Mock.js.Mocker.createInstancePropertyDescriptorListener (http://localhost:9876/_karma_webpack_/vendor.bundle.js:162226:27)
	    at Object.get (http://localhost:9876/_karma_webpack_/vendor.bundle.js:162178:27)
	    at isUnconfigurable (http://localhost:9876/_karma_webpack_/polyfills.bundle.js:7951:22)
	    at Function.Object.defineProperty (http://localhost:9876/_karma_webpack_/polyfills.bundle.js:7914:13)
	    at Mocker.webpackJsonp../node_modules/ts-mockito/lib/Mock.js.Mocker.createInstancePropertyDescriptorListener (http://localhost:9876/_karma_webpack_/vendor.bundle.js:162229:16)
	    at Object.get (http://localhost:9876/_karma_webpack_/vendor.bundle.js:162178:27)
	    at isUnconfigurable (http://localhost:9876/_karma_webpack_/polyfills.bundle.js:7951:22)
	    at Function.Object.defineProperty (http://localhost:9876/_karma_webpack_/polyfills.bundle.js:7914:13)
	    at Mocker.webpackJsonp../node_modules/ts-mockito/lib/Mock.js.Mocker.createInstancePropertyDescriptorListener (http://localhost:9876/_karma_webpack_/vendor.bundle.js:162229:16)` 

but it's fine when running the tests in Windows 10.

**Comment by NagRock** [comment]
@realgsong thanks for reporting this. Should be fixed in version 2.2.9. I will mark 2.2.8 version as deprecated.
